wp-tab-widget.php

```diff
@@ -344,7 +345,7 @@ public function ajax_wpt_widget_content() {
 			}
 			$show_thumb = isset( $args['show_thumb'] ) ? $args['show_thumb'] : 0;
 			$thumb_size = isset( $args['thumb_size'] ) ? $args['thumb_size'] : 'small';
-			if ( $thumb_size != 'small' && $thumb_size != 'large' ) {
+			if ( $thumb_size != 'small' && $thumb_size != 'large' && $thumb_size != 'medium' ) {
```

Comment:
Since there are many strings to compare, can you please to use `in_array()` instead?

Reply:
Does it even make sense to check for this condition fragment `$thumb_size != 'small'`? Unless the value is `medium` or `large` we can default to `small`, right?

Reply:
@aidik Yes. You are right. Thank for pointing out.